src/libs/subcategories/__mocks__/useSubcategories.ts

```diff
@@ -0,0 +1,9 @@
+import { subcategoriesDataTest } from 'libs/subcategories/fixtures/subcategoriesResponse'
+import { useSubcategories as ActualUseSubcategories } from 'libs/subcategories/useSubcategories'
```

Comment:
pour quelle raison mettre une majuscule ?

```suggestion
import { useSubcategories as actualUseSubcategories } from 'libs/subcategories/useSubcategories'
```